java/ql/src/experimental/Security/CWE/CWE-094/InjectionLib.qll

```diff
@@ -0,0 +1,14 @@
+import java
```

Comment:
`InjectionLib` is an odd name for a file containing only a taint step traversing getters. Given it's query-local perhaps just `Utils` or `FlowUtils`?

Reply:
Sure, I renamed it to `FlowUtils`.